tests/calculations/test_cp.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""Tests for the `PwCalculation` class."""
+from aiida.common import datastructures
+import pytest
+
+
+@pytest.mark.parametrize('autopilot', [True, False])
+def test_cp_autopilot(
+    aiida_profile, fixture_sandbox, generate_calc_job, generate_inputs_cp, file_regression, autopilot
+):
+    """Test a default `PwCalculation`."""
+    entry_point_name = 'quantumespresso.cp'
+
+    inputs = generate_inputs_cp(autopilot=autopilot)
+    calc_info = generate_calc_job(fixture_sandbox, entry_point_name, inputs)
+    upf = inputs['pseudos']['Si']
+
+    cmdline_params = ['-in', 'aiida.in']
+    local_copy_list = [(upf.uuid, upf.filename, './pseudo/Si.upf')]
+    retrieve_list = [
+        'aiida.out', './out/aiida_51.save/data-file-schema.xml', './out/aiida_51.save/data-file.xml', './out/aiida.cel',
+        './out/aiida.con', './out/aiida.eig', './out/aiida.evp', './out/aiida.for', './out/aiida.nos',
+        './out/aiida.pol', './out/aiida.pos', './out/aiida.spr', './out/aiida.str', './out/aiida.the',
+        './out/aiida.vel', './out/aiida.wfc', './out/aiida_51.save/print_counter',
+        './out/aiida_51.save/print_counter.xml'
+    ]
+    retrieve_temporary_list = [['./out/aiida.save/K*[0-9]/eigenval*.xml', '.', 2]]
+
+    # Check the attributes of the returned `CalcInfo`
+    assert isinstance(calc_info, datastructures.CalcInfo)
+    assert sorted(calc_info.cmdline_params) == sorted(cmdline_params)
```

Comment:
This was a bug in previous tests, this should become something like
`assert sorted(calc_info.codes[0].cmdline_params) == sorted(cmdline_params)`
(it's in the code_info)

Reply:
modified in commit f2f78da3